src/json/brace-data-exchange.schema.json

```diff
@@ -1636,6 +1667,31 @@
           "title": "Environmental Problems",
           "description": "The property has problems that are environmental in nature (e.g. black mold.)"
         },
+        "has_new_principal_residence_over_100mi_away": {
+          "$ref": "#/definitions/tribool",
+          "title": "Has New Principal Residence Over 100 Miles Away",
+          "description": "The borrower or co-borrower has established a new principal residence more than 100 miles from the borrower’s current, or former, principal residence. "
+        },
+        "has_new_dependents_property_doesnt_meet_needs": {
+          "$ref": "#/definitions/tribool",
+          "title": "Has New Dependents property Doesn't Meet Needs",
```

Comment:
This is super nitpicky but could we capitalize `property` here?

Reply:
Done.